src/managing-workflow/tuning-component-settings.md

```diff
@@ -0,0 +1,115 @@
+GUNICORN_WORKERS                                | number of [gunicorn][] workers spawned to process requests (default: num_cpus * 2 + 1)
+RESERVED_NAMES                                  | a comma-separated list of names which applications cannot reserve for routing (default: "deis")
+SLUGRUNNER_IMAGE_NAME                           | the image used to run buildpack application slugs (default: "quay.io/deisci/slugrunner:canary")
+SLUG_BUILDER_IMAGE_PULL_POLICY                  | the kubernetes [image pull policy][pull-policy] for slugbuilder (default: "Always")
+DOCKER_BUILDER_IMAGE_PULL_POLICY                | the kubernetes [image pull policy][pull-policy] for dockerbuilder (default: "Always")
+DEIS_DEPLOY_BATCHES                             | the number of pods to bring up and take down sequentially during a scale (default: all)
+KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS | how many seconds kubernetes waits for a pod to finish work after a SIGTERM before sending SIGKILL (default: 30)
+
+## Customizing the Database
+
+The following environment variables are tunable for the [Database][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+BACKUP_FREQUENCY  | how often the database should perform a base backup (default: "12h")
+BACKUPS_TO_RETAIN | number of base backups the backing store should retain (default: 5)
+
+## Customizing Fluentd
+
+The following environment variables are tunable for [Fluentd][logger]:
+
+Setting           | Description
+----------------- | ---------------------------------
+SYSLOG_HOST_1     | The hostname of a remote syslog endpoint for shipping logs
+SYSLOG_PORT_1     | The port of a remote syslog endpoint for shipping logs
+
+## Customizing the Logger
+
+The following environment variables are tunable for the [Logger][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+STORAGE_ADAPTER   | How to store logs that are sent to the logger (default: "memory")
+NUMBER_OF_LINES   | How many lines to store in the ring buffer (default: 1000)
+
+## Customizing the Monitor
+
+The monitor component uses [Telegraf](https://github.com/influxdata/telegraf) under the hood, and
+derives most of its configuration from it. Please see
+[telegraf configuration](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md)
+for more information on tuning the [Monitor][] component.
+
+## Customizing the Registry
+
+The [Registry][] component can be tuned by following the
+[deis/distribution config doc](https://github.com/deis/distribution/blob/master/docs/configuration.md).
+
+## Customizing the Router
+
+The following environment variables are tunable for the [Router][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+POD_NAMESPACE     | The pod namespace the router resides in. This is set by the [Kubernetes downward API][downward-api].
+
```

Comment:
It might be worth noting here that the majority of router settings are tunable through annotations instead, which allows the router to be re-configured with zero downtime post-installation. A link to the router's own documentation might be helpful. It's far too extensive to summarize here.